### DISCUSSION TITLE
Documentation on how to connect to postgres from Azure Data Studio

### DISCUSSION THREAD

**Discussion by daskabe** [mediagis/nominatim-docker#542] [General]
**Is your feature request related to a problem? Please describe.**
Unable to connect to postgres db from Azure Data Studio
![image](https://github.com/mediagis/nominatim-docker/assets/94478096/454d3925-0fbc-4550-b5bc-d510abe6f599)

```
connection failed: could not receive data from server: Connection refused
```
**Describe the solution you'd like**
documentation on how to connect to postgres db from Azure Data Studio

**Describe alternatives you've considered**
i tried using the terminal to perform sql queries. 

**Additional context**
I am trying to do custom search as its not clear how to do it using the search api

**Comment by daskabe** [comment]
This worked for me :
```
docker run -it \
  -e PBF_URL=https://download.geofabrik.de/europe/monaco-latest.osm.pbf \
  -e REPLICATION_URL=https://download.geofabrik.de/europe/monaco-updates/ \
  -p 8080:8080 \
  -p 5432:5432 \ 
  --name nominatim \
  mediagis/nominatim:4.3 
```
And then try this in azure data studio to connect. The default password is in the docs.

<img width="491" alt="image" src="https://github.com/mediagis/nominatim-docker/assets/94478096/3d583ee1-7063-4918-8ffa-4fa595cf249e">

**Comment by leonardehrenfried** [comment]
Looks like you're on Windows. Persumably "localhost" doesn't work there and you will have to figure out how to address the container. Maybe by IP address?
